schedule.md

```diff
@@ -0,0 +1,16 @@
+# Schedule
+
+| Time                | Topic                     | Presenter        |
+| :---:               |    :----:                 |    :---:         |
+| 09:00 AM - 09:15 AM | Open Radar Science        | Max Grover       |
+| 09:30 AM - 10:00 AM | xradar Introduction       | Alfonso Ladino   |
+| 10:00 AM - 10:30 AM | Coffee break              |                  |
+| 10:30 AM - 11:00 AM | Py-ART/wradlib            | Kai Mühlbauer + Bobby Jackson |
+| 11:00 AM - 11:30 AM | Pyrad Introduction        | Jordi Figueras i Ventura + Daniel Wolfensberger    |
+| 12:00 PM - 12:30 PM | LROSE                     | Brenda Javornik + Jen DeHart |
+| 12:30 PM - 01:30 PM | Lunch                     |                              |
```

Comment:
I've talked to @ghiggi and we might need 10 to 15 Minutes for a short intro to gpm-api. Not sure where to squeeze this in.

Reply:
Can we include that in the project pitches in the afternoon? Or was there interest specifically in the morning?

Reply:
It can be in the afternoon, but half an hour and four workflows seems a bit tight. Or LROSE and PyRad do their workflow advertisement in the morning?

Reply:
Let's do 15 minutes after BALTRAD - then we can do a "lightning" style for the project pitches.  Everyone should advertise their workflow/project ideas throughout the tutorials earlier in the day!